chainerx_cc/chainerx/backward.cc

```diff
@@ -650,7 +650,7 @@ std::vector<absl::optional<Array>> Grad(
 
     BackwardImpl{inputs, outputs, actual_backprop_id, double_backprop, std::move(array_node_grad_map), retain_grad}.Run();
 
-    for (size_t i = 0; i < input_grads.size(); ++i) {  // NOLINT(modernize-loop-convert)
+    for (size_t i = 0; i < input_grads.size(); ++i) {
```

Comment:
Hm, why did you remove `NOLINT`?

Reply:
I mistook the loop, it was the first one of the method, which I rewrote to use the for each notation.
I cross-checked this running clang-tidy in my environment.

Reply:
Oh, I see!